### PR TITLE
feat(services-activities): Activities service setup

### PR DESCRIPTION
**Pull request by saevarma**
# Activities service

[Notion task](https://www.notion.so/Setup-Activities-service-fc33f411c43f49e1af963696c3a672f6)

## What

Initial setup for activities service. That will be iterated in further PRs.

## Why

To merge smaller iterations into main for testing on dev.

## Screenshots / Gifs

N/A

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Comment by saevarma** [comment]
@pshomov this is a new service that needs docker repository, postgres db and general infra review as it should be only available on internal network from our graphql server.

**Review by valurefugl — APPROVED** [review]

File: apps/services/activities/sequelize.config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+module.exports = {
+  development: {
+    username: 'dev_db',
+    password: 'dev_db',
+    database: 'dev_db',
+    host: 'localhost',
+    dialect: 'postgres',
+    port: 5434,
+  },
+  production: {
```
Comment (busla) [inline apps/services/activities/sequelize.config.js:11]: I assume you need production envs to be provisioned by devops, right?

**Review by busla — CHANGES_REQUESTED** [review]
LGTM

File: apps/services/activities/sequelize.config.js (hunk 1 above)
Comment (saevarma) [reply apps/services/activities/sequelize.config.js:11]: yup :)

Comment (saevarma) [reply apps/services/activities/sequelize.config.js:11]: @busla will you let me know when the envs have been provisioned?

I probably need to add grant for islandis and identity-server namespaces.

**Comment by saevarma** [comment]
Closing this as it is part of #9731

**Pull request closed without merging**